### DISCUSSION TITLE
Deploy database objects without creating Roles?

### DISCUSSION THREAD

**Discussion by minghungcho** [littleK0i/SnowDDL#2] [Q&A] [answered]
Is there an option to instruct SnowDDL to deploy database objects without creating roles and granting privileges?

**Comment by littleK0i** [comment, marked as answer]
Thank you for your interest in SnowDDL.

Yes, it is possible. For CLI interface, you may use option `--exclude-object-types` or `--include-object-types`.

For example, to resolve ONLY databases and schemas: 
```
snowddl <options> --include-object-types=DATABASE,SCHEMA apply
```

To resolve everything, EXCEPT roles and grants:
```
snowddl <options> --include-object-types=ROLE apply
```

Object type names and execution order is described on this documentation page: https://docs.snowddl.com/guides/object-types

Full list of SnowDDL options is documented on this page: https://docs.snowddl.com/basic/cli

Also, it is available with `snowddl --help` CLI command.
